### DISCUSSION TITLE
BaseRepurchase missing balance_security_id

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#244] [General]
We have resulting_security_ids which doesn't make sense for a repurchase, I think this was meant to be balance_security_id to represent the remainder for a partial repurchase

**Comment by pjohnmeyer** [comment]
TWG Notes

A full repurchase would have no balance security.

A partial repurchase would return some to the pool and the remainder would be allocated to the same stakeholder, so per our usual pattern this should be a `balance_security_id` not `resulting_security_ids`.
